submissions/satisfactorio/satisfactorio.md

```diff
@@ -0,0 +1,91 @@
+## Welcome to Satisfactorio!
+
+### What?
+
+**Satisfactorio** is an overhaul mod that bridges the gap between Satisfactory and Factorio. Every feature, from Milestones to Drones and everything in between, has been carefully re-created in Factorio.
+
+### Why?
+
+Modding isn't about "why", it's about "why not"! I enjoyed both Factorio and Satisfactory, and over on the Satisfactory mod portal there's a mod that attempts to bring Factorio elements into Satisfactory. Things like Inserters, Science Packs and other small scale modifications. That seemed like a cool concept so I wondered if maybe I could bring Satisfactory elements into Factorio. This [has been attempted](https://mods.factorio.com/mod/Satisfactory) before, however development doesn't seem to have gone particularly far before it was abandoned.
+
+In particular I was interested in how Satisfactory conveyor belts connect directly to input/output slots on machines. This also applies to train stations, where a train pulls in and dumps its cargo in the platform storage, to then be loaded onto a belt. Certainly a lot simpler than some of the optimised loading/unloading setups I've seen in Factorio!
+
+### Who?
+
+Just for a little background, I'm quite new to the modding scene. In fact, aside from a couple of very small mods (one to [calculate solar panel ratios](https://mods.factorio.com/mod/SolarRatio) and one to [copy a snapshot of signals](https://mods.factorio.com/mod/PasteSignals) from power poles to a constant combinator), I've never attempted any signifiant modding efforts.
+
+I am, at least, a software developer. My day-job is working on a [free-to-play Pokémon fan-game](https://pokefarm.com/), which I've been doing for the past 12 years now. But even then, that's all done in PHP and JavaScript. Lua, the language mods are written in, was completely new to me. Fortunately, the general problem-solving skills carried over quite well, so it was mostly a matter of learning the syntax and structure of Lua scripts.
```

Comment:
Wasn't learning the Factorio engine and API and what you can achieve with it a larger undertaking than learning Lua itself?